integration-cli/docker_api_info_test.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestInfoApi(t *testing.T) {
+	endpoint := "/info"
+
+	if _, err := sockRequest("GET", endpoint, nil); err != nil {
```

Comment:
`sockRequest` changed signature in master. Now you can get `StatusCode` and check it. Also maybe makes sense to unmarshall body to some temp structure for checking that important fields is in there.